packages/frontend/pages/MessageBoard.tsx

```diff
@@ -0,0 +1,57 @@
+import React, { useState, useRef, useEffect} from 'react'
+import { v4 as uuidv4 } from 'uuid'
+
+export function Message( {message} ) {
+    return (
+        <div>
+            <h4>Name: {message.name}</h4>
+            <p>Message:</p>
+            <p>{message.msg}</p>
+            <p>[DEBUG]MessageID: {message.id}</p>
```

Comment:
I suggest adding a class or toggle for hiding or disabling debug info

Reply:
It's only there for debug purposes, once its time to add classes it can be removed

Reply:
This can be removed as far as I know.

Reply:
It should be removed. Debug statements are only meant for local development and shouldn't end up in the code base